docker-compose-local.yml

```diff
@@ -26,7 +26,7 @@ services:
     depends_on:
       - discoveryservice
       - rabbitmq
-    image: aista/gmaps-adapter
+    build: ../gmaps-adapter
```

Comment:
This convention doesn't really work for most people's file structure, because we have to have this project in the gopath, and other projects are normally somewhere else.

Reply:
changed back to aista/gmaps-adapter